lte/gateway/c/core/oai/test/amf/amf_app_test_util.cpp

```diff
@@ -48,6 +48,43 @@ imsi64_t send_initial_ue_message_no_tmsi(
   return imsi64;
 }
 
+/* Create initial ue message without TMSI and replace tmsi in hexbuf */
```

Comment:
Use single line comment // instead of /**/

Reply:
done